most-run/test/setup.ts

```diff
@@ -39,15 +38,15 @@ describe('setup', function() {
       other: Stream<string>;
     };
 
-    function app(sources: MySources): MySinks {
+    function app(_sources: MySources): MySinks {
```

Comment:
Why the renaming?

Reply:
Because of variable name shadowing, see some lines below this

Reply:
I see